src/lib.rs

```diff
@@ -299,6 +314,56 @@ impl Uuid {
             _ => None,
         }
     }
+    
+    /// Creates a new `Uuid` (version 1 style) using a time value + seq + NodeID
+    ///
+    /// This expects two values representing a monotonically increasing value 
+    /// as well as a unique 6 byte NodeId, and a `UuidV1Context`
+    /// This function is only guaranteed to produce unique values if the following
+    /// conditions hold: 
+    /// 1. The NodeID is unique for this process.
+    /// 2. The Context is shared across all threads which are generating V1 UUIDs
+    /// 3. The supplied time value is monotonically increasing.
+    ///
+    /// The function is not guaranteed to produce monotonically increasing values
+    /// however.  There is a slight possibility that two successive equal time values
+    /// could be supplied and the sequence counter wraps back over to 0. 
+    ///
+    /// If uniqueness and monotonicity is required, the user is responsibile for ensuring
+    /// that the time value always increases between calls. 
+    /// (including between restarts of the process and device)
+    #[cfg(feature = "v1")]
+    pub fn new_v1(context: &UuidV1Context, time: u64, time_frac: u32, node: &[u8]) -> Result<Uuid, ParseError> {
```

Comment:
Perhaps `time_frac` could be `nanos: u32`? That way we could avoid floating math and such and it'd be a little clearer that this is a seconds/nanos pair.

Additionally, I think there's length restrictions on `node`, right? Could those be documented?

And finally, is it possible for this method to return an error? I figured it'd always generate a valid `Uuid`. If it can return an error though, can you document under what circumstances an error is returned?